src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,8 +1,51 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
 
     public void getStatistic(String fromFileName, String toFileName) {
-
+        File informationFromFile = new File(fromFileName);
+        File resultFile = new File(toFileName);
+        int supplySummary = 0;
+        int buySummary = 0;
+        int operationResult = 0;
+        StringBuilder stringBuilder = new StringBuilder();
+        try {
+            FileReader fileReader = new FileReader(informationFromFile);
+            BufferedReader bufferedReader = new BufferedReader(fileReader);
+            while (bufferedReader.ready()) {
+                stringBuilder.append(bufferedReader.readLine()).append(",");
+            }
+        } catch (IOException e) {
+            throw new RuntimeException("Some went wrong", e);
+        }
+        String[] datesFromFile = stringBuilder.toString().split(",");
+        for (int i = 0; i < datesFromFile.length; i++) {
+            if (datesFromFile[i].equals("supply")) {
```

Comment:
let's make constants for operation types, dividers and word positions (operation,amount)